src/util.cpp

```diff
@@ -120,22 +120,28 @@ std::vector<bool> hex_to_binary_vector(std::string hex_str)
 // Takes an hexadecimal digest and converts it into a binary vector
 std::vector<bool> hex_digest_to_binary_vector(std::string hex_str)
 {
-    if (hex_str.length() != ZETH_DIGEST_HEX_SIZE) {
-        throw std::length_error(
-            "Invalid string length for the given hex digest (should be "
-            "ZETH_DIGEST_HEX_SIZE)");
-    }
-
     return hex_to_binary_vector(hex_str);
 }
 
 bits256 hex_digest_to_bits256(std::string str)
 {
-    return get_bits256_from_vector(hex_digest_to_binary_vector(str));
+    if (str.length() != 64) {
+        throw std::length_error(
+            "Invalid string length for the given hex digest (should be "
+            "64)");
+    }
+
+    return get_bits256_from_vector(hex_to_binary_vector(str));
 }
```

Comment:
Just realized that in https://github.com/clearmatics/zeth/blob/packing/src/test/note_test.cpp we do: 
```
get_bits256_from_vector(hex_digest_to_binary_vector("FF0000000000000000000000000000000000000000000000000000000000000F"));
```

and
```
bits64 value_bits64 = get_bits64_from_vector(hex_to_binary_vector("2F0000000000000F"));
```

Let's just replace `get_bits64_from_vector(hex_to_binary_vector(` by `hex_value_to_bits64`
and `get_bits256_from_vector(hex_digest_to_binary_vector(` by `hex_digest_to_bits256`

Reply:
Note: This can be fixed in another PR as it is not strictly related to this one

Reply:
Fixed in PR #151